src/atoms/Button/__tests__/button.spec.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Button from '../index';
```

Comment:
this relative import should go after the absolute imports; the linter will complain